packages/client/mutations/AddCommentMutation.ts

```diff
@@ -47,12 +47,15 @@ export const addCommentMeetingUpdater: SharedUpdater<AddCommentMutation_meeting>
     addNodeToArray(comment, store.get(threadParentId), 'replies', 'threadSortOrder')
     return
   }
-  const reflectionGroupId =
-    threadSource === ThreadSourceEnum.REFLECTION_GROUP ? comment.getValue('threadId') : undefined
-  if (reflectionGroupId) {
-    const reflectionGroup = (reflectionGroupId && store.get(reflectionGroupId as string)) || null
-    const reflectionGroupConn = getReflectionGroupThreadConn(reflectionGroup)
-    safePutNodeInConn(reflectionGroupConn, comment, store, 'threadSortOrder', true)
+  const threadSourceId =
```

Comment:
i like how you kept the same base logic in place here. Looking at this again, any `threadable` type that has a `threadSource` is guaranteed to have a `threadId`, so we could simplify this to:
`const threadSourceId = comment.getValue('threadId')`